core/src/OpenMP/Kokkos_OpenMP_Instance.hpp

```diff
@@ -128,11 +128,6 @@ class OpenMPInternal {
   void print_configuration(std::ostream& s) const;
 };
 
-inline bool execute_in_serial(OpenMP const& space = OpenMP()) {
-  return (space.impl_internal_space_instance()->get_level() < omp_get_level() &&
-          !(omp_get_nested() && (omp_get_level() == 1)));
-}
-
```

Comment:
Keep these lines here instead (taking into account what's on `develop`).

Reply:
Those lines represent a second definition, and had to be deleted.